docs/demos/tutorials/02_Exploratory_analysis.ipynb

```diff
@@ -2619,23 +2619,23 @@
   {
    "attachments": {},
    "cell_type": "markdown",
-   "id": "1f37cb1e",
+   "id": "c537f065-f61f-402a-ad2c-a8575b6207ab",
```

Comment:
It looks as if the graphs have the `<vegalite 5 object>` string baked into all of the graphs outputs above.

Reply:
Fixed - still had the reneder as "mimetype" by mistake